sky/client/common.py

```diff
@@ -206,9 +206,14 @@ def _upload_chunk_with_retry(params: UploadChunkParams) -> None:
                     f'Retrying... ({attempt + 1} / {max_attempts})')
                 time.sleep(1)
             else:
+                try:
+                    response_details = response.json().get('detail')
```

Comment:
Do we need this for other `.json()` calls?

Reply:
Only for the middleware I think. This will call .json() indiscriminately on all requests, but usually we only call it on requests that are expected to be json (which is fine).